docs/0002-atlas-translations-management.rst

```diff
@@ -0,0 +1,136 @@
+Atlas Translations Management Design
+####################################
+
+Date: 25 March 2024
+
+Status
+******
+Pending
+
+Context
+*******
+
+Open edX microservices and micro-frontends is using `OEP-58`_ to pull the latest
+translations from the `openedx-translations repository`_.
+
+The gist of `OEP-58`_ is two processes:
+
+- Extract the English translation source entries and commit them into the `openedx-translations repository`_
+- Provide a command to pull translations in both development and web server image builds
+
+Microservices, micro-frontends, XBlocks and plugins uses one or two files which makes it easier for Translators
+to locate and translate relevant resources.
+
+The `edx-platform`_ used to have 7 different resources, but this has been deprecated in favor of ``edx-platform.po``
+and ``edx-platform-js.po`` files. The details in the `edx-platform resources combination decision`_.
+
+The ``openedx-app-ios`` uses a clean architecture. Therefore, each app has its own i18n strings files. This is
+useful for Developer Experience, but we believe it's not good for Translator Experience.
+
+Decision
+********
+
+This decision will require few changes to the translation workflow in comparison with the old ``edx-app-ios``:
+
+- Translation files are no longer comitted directly in the mobie app ``openedx-app-ios/android`` repositories
```

Comment:
```suggestion
- Translation files are no longer comitted directly to the mobie app repositories (``openedx-app-ios``/``openedx-app-android``)
```

It might be nice to turn those repo names into links.

Reply:
```suggestion
- Translation files are no longer comitted directly to the mobie app repositories (`openedx-app-ios`_ / `openedx-app-android`_)
```